runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json

```diff
@@ -47,6 +47,12 @@
       "icon": 3410,
       "name": "Serum 207 (3)",
       "xp": 50
+    },
+	{
```

Comment:
Format this consistently with the rest of the file.

Reply:
@Adam- I believe this is just the diff showing the indenting incorrectly. I have looked at the file in its current state and looks fine? Unless i'm missing something...

Reply:
Its a tabs/spaces thing

Reply:
Done